### PR TITLE
Development script for quick and reliable setup

### PR DESCRIPTION
**Pull request by redtachyons**
Setup script for automating development setup

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by redtachyons** [comment]
Please review @publiclab/reviewers

**Comment by jywarren** [comment]
Wow! This is really cool! @publiclab/reviewers -- esp. @ryzokuken wanted to have a makefile -- what do you think?

**Comment by jywarren** [comment]
And thank you very much!!

**Comment by ryzokuken** [comment]
@jywarren I think we had merged the Makefile, it's already in the codebase. We just didn't end up making changes to the README 😅

Meanwhile, I'll take a look at this.

**Comment by redtachyons** [comment]
Just noticed Makefile now !. And seeing a Makefile in the ruby project for the first time.
bin/setup is now a rails convention and included by default in Rails 5. So I think bin/setup is better here

**Comment by ryzokuken** [comment]
@redtachyons I agree. I think it's cool if you removed the Makefile in this PR as well, it's not used anyway and just seems redundant when this gets merged.

File: bin/setup [hunk 1]
```diff
@@ -20,7 +20,7 @@ chdir APP_ROOT do
 
 
   puts "\n== Copying sample files =="
-  unless File.exist?('config/database.yml')
+  unless File.exist?('db/schema.rb')
     cp 'db/schema.rb.example', 'db/schema.rb'
```
Comment (seafr) [inline bin/setup]: Isn't this a shell command?

Comment (seafr) [reply bin/setup]: Same for line 28

Comment (tachyons) [reply bin/setup]: Yes, but these methods are part of ruby standard library as well. See https://ruby-doc.org/stdlib-2.2.2/libdoc/fileutils/rdoc/FileUtils.html

**Comment by tachyons** [comment]
@ryzokuken Sure, will do

File: bin/setup (hunk 1 above)
Comment (seafr) [reply bin/setup]: Ah, news to me! Thank you for this.

**Comment by jywarren** [comment]
Thanks, everybody!!! 👍

**Pull request merged**